templates/f5/0/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+external-lb:
+  image: rancher/external-lb:v0.1.1
+  command: -debug -provider=f5_BigIP
```

Comment:
@ibuildthecloud I kept debug since hardly any useful information printed on info during normal syncs of LB records, making it hard to debug when some setup has issues.